Makefile

```diff
@@ -66,10 +68,20 @@ test-clover:
 	docker build -f examples/clover/Dockerfile .
 
 publish:
-	$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-*"
+	$(AWS) s3 cp \
+	  --acl public-read \
+	  --exclude "*" \
+	  --include "test-reporter-$(VERSION)-*" \
```

Comment:
I think there's still the risk of overriding an existing version if someone forgets updating `$(VERSION)` before running `make release`